### ISSUE TITLE
combining bam files generated with bwa mem and vg for processing and analyses

### ISSUE THREAD

**Issue by mariels** [vgteam/vg#3036]
Hello,

I have a dataset of around 60 whole modern genomes mapped with bwa mem, and I also have four ancient genomes. I would like to use vg to remap my ancient data only, to limit reference bias.

I was wondering whether the bam files generated with vg would be directly comparable to the ones generated with bwa, i.e. if they will be indexed the same way. I would use the same reference genome to construct the variation graph. Then, I would like to apply the same filters using bed files, etc on the bam files generated with vg and bwa, and run analyses on all the modern (bwa) and ancient samples (vg) together.

Sorry if this question has already been asked or if I missed the information somewhere in the related papers/tutorials.

Many thanks,

Marie

**Comment by ekg** [comment]
Yes, they should be the same format and you should be able to mix them and apply the same tools to them as any normal BAM.

In any case, if you did map the modern samples with vg, I wouldn't map them using the same ultra-sensitive parameters suggested for ancient DNA. So unless you have lots of compute time available, there will be differences with a complete remap in vg. You will have much less reference bias around indels and clusters of SNPs though, so it isn't a bad idea. It's just costly because vg map is pretty slow (7-10x slower than bwa mem).

**Comment by glennhickey** [comment]
Also note: It is very important to use the current latest vg release (1.27.1) or newer if you are using vg to make BAMs. Previous versions had a bug that sometimes flipped the quality strings.

**Comment by ekg** [comment]
Yes! That's right!!

**Comment by mariels** [comment]
Great, many thanks for your answers and advices!

**Comment by faithokamoto** [comment]
Seems this was answered at the time.
